runtime/src/bank.rs

```diff
@@ -486,6 +481,25 @@ impl Bank {
         self.store_account(&sysvar::slot_hashes::id(), &account);
     }
 
+    fn update_epoch_stakes(&mut self, leader_schedule_epoch: Epoch) {
+        // update epoch_stakes cache
+        //  if my parent didn't populate for this staker's epoch, we've
+        //  crossed a boundary
+        if self.epoch_stakes.get(&leader_schedule_epoch).is_none() {
+            self.epoch_stakes
+                .insert(leader_schedule_epoch, self.stakes.read().unwrap().clone());
+
+            if leader_schedule_epoch >= MAX_LEADER_SCHEDULE_STAKES {
+                self.epoch_stakes
+                    .remove(&(leader_schedule_epoch - MAX_LEADER_SCHEDULE_STAKES));
+            }
+            error!(
```

Comment:
Why error?

Reply:
Yes, the log level should be lower than `error`.

Just to stand out for [testing purpose as said above](https://github.com/solana-labs/solana/pull/7098/files#r350474556). Sorry for misleading you!